posthog/warehouse/models/credential.py

```diff
@@ -0,0 +1,11 @@
+from posthog.models.utils import UUIDModel, CreatedMetaFields, sane_repr
```

Comment:
Follow up from our call, are we cool with credentials like this for MVP or should we have the ARN config flow for MVP atleast?

CC @fuziontech